test/src/api/commands/element/testClearValue.js

```diff
@@ -117,6 +118,45 @@ describe('clearValue', function() {
       client.start(done);
     });
   });
+ 
+  it('client.clearValue() with webdriver protocol - ensuring working of sendKeys()', function (done) {
+    Nightwatch.initClient({
+      selenium: {
+        version2: false,
+        start_process: false,
+        host: null
+      },
+      output: false,
+      silent: false,
+      webdriver: {
+        host: 'localhost',
+        start_process: false
+      }
+    }).then((client) => {
+      MockServer.addMock(
+        {
+          url: '/session/13521-10219-202/element/5cc459b8-36a8-3042-8b4a-258883ea642b/property/value',
+          response: {value: 'sampleText'}
+        },
+        true
+      );
```

Comment:
You should try running a test with the `.clearValue()` command locally (like the sample test given in the issue) with `--verbose` flag, and mock all the requests going out for `.clearValue()`, while ensuring that the command is working correctly.

Alternatively, you can also set `output` to `true` above, which will give you all the verbose logs for the test.